CITATION.md

```diff
@@ -0,0 +1,49 @@
+# Citing and logo
+
+## Citing pandas
+
+If you use *pandas* for a scientific publication, we would appreciate citations to the published software and one of the two given papers:
+
+- *pandas* version 1.0.1 published on Zenodo (please find us on Zenodo and replace with the citation for the version you are using)
+```
+@software{reback2020pandas,
+  author       = {The pandas development team},
+  title        = {pandas-dev/pandas: Pandas 1.0.1},
+  month        = feb,
+  year         = 2020,
+  publisher    = {Zenodo},
+  version      = {v1.0.1},
```

Comment:
Not sure, but may be instead of 1.0.1 we could have something like `REPLACE_BY_USED_VERSION`? Or may be we can simply remove the version and make things easier?